### ISSUE TITLE
playground query not hightlighting? 

### ISSUE THREAD

**Issue by archywillhe** [tree-sitter/tree-sitter#2518]
So this is what I see:

<img width="360" alt="Screenshot 2023-08-18 at 12 48 41 AM" src="https://github.com/tree-sitter/tree-sitter/assets/10269911/c944a270-a4c3-426f-b830-bf1c83d23b31">

Can there be a bug somewhere? Or I'm not seeing the color?

**Comment by archywillhe** [comment]
related: https://github.com/tree-sitter/tree-sitter/issues/1714

**Comment by archywillhe** [comment]
oh I need to `(comment) @file`. sorted!

**Comment by v-p-b** [comment]
@archywillhe saved my day by linking #1714, thanks!. I never had problem with seeing colors, and have a drivers license, yet I can see no highlight without Accessibility=on. I tried this with Chrome and Firefox with dark and light mode, highlight was not visible in any of the cases.

I think we should either reopen this issue or create a new one for fixing the highlight styling.

**Edit:** I _just now_ realized that it is text color that changes!! I swear I couldn't tell after numerous tries. I noticed only after tried to look at styles using devtools. This must be some of those effects that are used in magic tricks!
